Frontend/src/common/interfaces/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { View, TouchableOpacity, Text, StyleSheet } from "react-native";
+
+interface PaginationProps {
+  currentPage: number;
+  showNextPage: boolean;
+  goToPreviousPage: () => void;
+  goToNextPage: () => void;
+}
+
+const Pagination: React.FC<PaginationProps> = ({
+  currentPage,
+  showNextPage,
+  goToPreviousPage,
+  goToNextPage,
+}) => {
+  return (
+    <View style={styles.pageContainer}>
+      <TouchableOpacity
+        style={styles.pageButton}
+        onPress={goToPreviousPage}
+        disabled={currentPage === 0}
+      >
+        <Text style={styles.pageButtonText}>
+          {"<<"} Pág {currentPage}{" "}
+        </Text>
+      </TouchableOpacity>
+      <TouchableOpacity
+        style={styles.pageButton}
+        onPress={goToNextPage}
+        disabled={showNextPage}
```

Comment:
Keep in mind that when the condition of `disabled` is `true` then the button is disabled. So when `showNextPage===true` then this button will be disabled. A more fitting name for the variable would be `isAtLastPage`, for example.